programs/intent-transfer/src/lib.rs

```diff
@@ -86,6 +86,7 @@ pub struct RegisterNttConfig<'info> {
     pub ntt_manager: UncheckedAccount<'info>,
 
     /// this account's address is verified to make sure it corresponds to this program
+    #[account(seeds = [crate::ID.as_ref()], bump, seeds::program = bpf_loader_upgradeable::ID)]
```

Comment:
@anihamde I think this should be included in your commit as per @guibescos 's feedback but I added it here so I could write the scripts without manually specifying this account